.travis.yml

```diff
@@ -27,6 +27,8 @@ script:
   - 'if [ "$TRAVIS_PULL_REQUEST"  = "false" ]; then make images; fi'
   # Process 'make images' when running as a pull request
   - 'if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then unset quay_repository && make images; fi'
+   # Process 'make push-images' when NOT a pull request
```

Comment:
Is it correct?
I though that the image actually needs to be pushed in case of the PR, so KinD on Travis can pull it and use it in tests.

If I get it right the cases when Travis is running:
- PR check - needs image
- Build triggered by merged commit in repo - needs image
- Release build - doesn't need image

Reply:
I guess I was wrong, PR check passed.